challenges/test_binary_search.py

```diff
@@ -0,0 +1,39 @@
+
+
+import binary_search
+
+import pytest
+
+
+def test_sorted_array_is_empty():
+    """
+    test if sorted array is empty
+    """
+    sorted_array = []
+    test_value = 2
+    assert binary_search(sorted_array, test_value)
```

Comment:
Currently this test only checks if the `binary_search` function returns anything at all. You should always check if the value being returned is the correct value, like in your other tests.